server/lib/models/topic.go

```diff
@@ -1,7 +1,15 @@
 package models
 
 import (
-	proto "../../proto"
+	"log"
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"math/rand"
+
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/yue"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/yue (can't find import: github.com/umegaya/yue)